pkg/standalone/run.go

```diff
@@ -73,7 +73,7 @@ type componentMetadataItem struct {
 	Value string `yaml:"value"`
 }
 
-func getDaprCommand(appID string, daprHTTPPort int, daprGRPCPort int, appPort int, configFile, protocol string, enableProfiling bool, profilePort int, logLevel string, maxConcurrency int, placementHost string) (*exec.Cmd, int, int, int, error) {
+func getDaprCommand(appID string, daprHTTPPort int, daprGRPCPort int, appPort int, configFile, protocol string, enableProfiling bool, profilePort int, logLevel string, maxConcurrency int, placementHost string, componentsPath string) (*exec.Cmd, int, int, int, error) {
```

Comment:
is componentsPath always current directory ?

Reply:
its set here https://github.com/dapr/cli/pull/367/files#diff-184e85420a4841e42d77c23247296911R273

Reply:
I don't see where it is set, @amanbha. Trace the call from from cmd/run

Reply:
We need to make sure `componentsPath` is set to the default path when it's passed to this method.

Reply:
This is missing a default:

https://github.com/dapr/cli/blob/c37118166b1b6c869cd2795863eb99f4960fe100/cmd/run.go#L248

Reply:
We're telling the user there's a default  path, but its not being set.

Reply:
The default dir `getDefaultComponentsFolder()` should be set as the default value for the flag here.